queue-question/index.js

```diff
@@ -0,0 +1,49 @@
+
+const { QueueServiceClient, StorageSharedKeyCredential } = require("@azure/storage-queue");
+
+require("dotenv").config();
+module.exports = async function (context, req) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+
+    if (req.body && req.body.questionTitle) {
+        await enqueue(context, req.body)
+    }
+    else {
+        context.res = {
+            status: 400,
+            body: "Please pass a name on the query string or in the request body"
+        };
+    }
+}
+
+ async function enqueue(context, question) {
```

Comment:
This is our most complex function so it's probably worth having unit tests for this?